cocktails9/cocktails9/Views/loginView.swift

```diff
@@ -0,0 +1,134 @@
+import SwiftUI
+
+
+struct User: Codable {
+    let email: String
+    let password: String
+}
+
+struct CustomTextFieldStyle: TextFieldStyle {
+    
+    let icon: String
+    
+    func _body(configuration: TextField<_Label>) -> some View {
+        HStack {
+            Image(systemName: icon)
+                .foregroundColor(.gray)
+            configuration
+        }
+        .padding()
+        .background(Color.gray.opacity(0.2))
+        .cornerRadius(8)
+    }
+}
```

Comment:
This needs to go to a different file

Reply:
Just saw that this is covered with #3 . You can wait for it, or cherry-pick this file from Login commit. Just make sure to name it the same and keep it the same to avoid conflicts